pkg/nsx/services/inventory/compare.go

```diff
@@ -62,6 +64,25 @@ func compareContainerApplicationInstance(pre interface{}, cur interface{}, prope
 	}
 }
 
+func compareContainerIngressPolicy(pre interface{}, cur interface{}, property *map[string]interface{}) {
```

Comment:
Not adding by this PR, I just notice maybe there is no need to use pointer for map, i.e. we can also update the map in function by passing `property map[string]interface{}`? As we have the plan to have a code refactor in the future, we can take this into consideration at that time.

Reply:
+1